apigentools/cli.py

```diff
@@ -26,6 +26,12 @@ def get_cli_parser():
         default=env_or_val('APIGENTOOLS_GIT_VIA_HTTPS', False, __type=bool),
         help='Use HTTPS for interacting with the git repositories. Otherwise use SSH.'
     )
+    p.add_argument(
+        '--git-via-https-oauth-token',
+        action='store_true',
```

Comment:
Would this really work with `store_true` and `default` specified? If you really want to use this as just a flag (IOW not accept the oauth token on commandline, which I guess is fine), I think you should do:

```
p.add_argument(
        '--git-via-https-oauth-token',
        action='store_const',
        const=env_or_val('APIGENTOOLS_GIT_VIA_HTTPS_OAUTH_TOKEN', None),
        help='Insert OAuth token in the repo URL when using HTTPS for interacting with the git repositories.'
    )
```

That will effectively store the `const` value in `args.git_via_https_token` if the flag is specified, otherwise it will store `None`. AFAICS what you have will actually store `True` as value if you specify the flag.

Reply:
My bad, that was a copypaste leftover, I don't think the arg requires any action